<ui version="4.0" >
 <class>RecordForm</class>
 <widget class="QewExtensibleDialog" name="RecordForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>29</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="recordCheckBox" >
     <property name="text" >
      <string>Record</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLineEdit" name="fileLineEdit" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="fileNameButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection" >
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QewExtensibleDialog</class>
   <extends>QDialog</extends>
   <header>/usr/include/qew5/qewextensibledialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../sim/imagenes/sim-icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>recordCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fileLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>26</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel" >
     <x>281</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recordCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fileNameButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>76</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel" >
     <x>779</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
